server/configs/test.conf

```diff
@@ -6,6 +6,8 @@ listen: 127.0.0.1:4242
 
 http: 8222
 
+base_path: /nats
```

Comment:
Sorry to notice that now, but should we call it `http_base_path` or `http_root` or something like that but with leading `http_` to make it clear that this is a http option?

Reply:
Good you mentioned, because I was also struggling with that, I was not really sure.
I kind of started with `http_` because the monitoring part starts with `http_`. But then I started to notice that you have other http handlers (like profiling), that my base path doesn't include them, in the end I decided to remove the prefix.

I'd use `http_base_path` other than `http_root` because I think it's more common in other projects.

Reply:
Let's go with `http_base_path` and HTTPBasePath for option, etc..